site/pages/theme-topic-en.hbs

```diff
@@ -0,0 +1,127 @@
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+			</div>
+		</div>
+	</section>
+	<section class="gc-features mrgn-tp-xl">
+		<div class="row row-eqht">
+			<div class="col-md-8">
+				<div class="row">
+					<div class="col-md-6 col-xs-12">
+						<img class="img-responsive mrgn-bttm-sm" src="{{assets}}/img/360x203.png" alt=""/>
+					</div>
+					<div class="col-md-6 col-xs-12">
+						<h3 class="h5"><a class="stretched-link" href="#">{{{i18n "de-title-link"}}}</a></h3>
+						<p>{{{i18n "de-description"}}}</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-md-4">
+				{{>follow-us}}
+			</div>
+		</div>
+	</section>
+	<section class="provisional gc-contributors">
+		<h2 class="small">From:</h2>
+		<ul class="small">
```

Comment:
Can you apply the "small" into the gc-contributors CSS? The "small" is the default style.